src/utils/collapsedRow/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ProcessedLogLine } from "types/logs";
+
+/**
```

Comment:
this function might belong better in `LogRow/utils.ts`!

Reply:
I think given the fact that it is also used by the filter util method it should remain in the utils folder. Its strange to import a function from a component folder for use in a generic util.